src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -318,8 +318,7 @@ def refreshCache(self):
 
             # now check for sites in drain and adjust the possible locations
             # also check if there is at least one site left to run the job
-            if len(possibleLocations) == 0:
-                newJob['name'] = loadedJob['name']
+            if len(possibleLocations) == 0:  # FIXME I don't see any of these 3 keys in the picke file?!?
```

Comment:
@ticoann Seangchan, I had a look at a couple job pickle files today and I couldn't find any of these keys. Am I missing any detail? Or should we just get rid of of them?

Reply:
It only happen a certain case. I don't remember exactly but when submit failed, Ops wanted to know where the jobs supposed to go.

https://github.com/dmwm/WMCore/commit/0920e9857d682cf511eeb5e645ce93382767c6f9#diff-e453fdf0929e6edc3c8e541f17fea9b7

Reply:
Thanks for the pointer! I'm removing my comment.